lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -154,7 +154,8 @@ export default function ContentSelector({
 
   const selectCanvasFile = (file: File) => {
     cancelDialog();
-    onSelectContent({ type: 'file', file });
+    // file.id is a URL with a `canvas://` prefix.
```

Comment:
These three functions are the same, we could replace them with `selectLMSFile` or similar.